src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx

```diff
@@ -208,6 +208,35 @@ Another great way to explore your data is to go into any existing dashboards you
   Charts built with NRQL will have **View query** as an option. You can then edit and customize that query to see how your changes affect the resulting visualization.
 </figcaption>
 
+## Comments [#comments]
+
+When writing a NRQL query, you can add comments to a line or section of text. Comments can help your team members better understand and use the queries you make. 
```

Comment:
What do you think about merging the two sentences here? "section of text that help your team members" Just a thought.